db/payment_icons.yml

```diff
@@ -941,4 +941,4 @@
 -
   name: bri_direct_debit
   label: BRI Direct Debit
-  group: other
+  group: other
```

Comment:
Can we drop this change?

Reply:
yes.

Reply:
Hmm change is still here. Can we `git checkout origin/master db/payment_icons.yml` and commit/push that?

Reply:
dropped the change. can you please check it ?

Reply:
What @larouxn meant is, since there are 0 changes in `db/payment_icons.yml` files, can we ignore this file and keep the original state?

Reply:
yes, we can keep the original state